### PR TITLE
update rewards msg types to match kava8, fix typo in swap deposit msg

### PR DESCRIPTION
**Pull request by downing034**

### CODE REVIEW

File: src/client/index.js [hunk 1]
```diff
@@ -888,16 +888,16 @@ class KavaClient {
 
   /**
    * Claim Hard protocol reward using a specific multiplier and denoms
-   * @param {String} multiplierName the multiplier to claim with, such as 'small' or 'large'
    * @params {Array} choose which denom(s) of your rewards that you would like to claim
+   * and at which multiplier you claim them
+   * [{ denom: 'hard', multiplierName: 'large'}, { denom: 'ukava', multiplierName: 'small' }]
```
Comment (downing034) [inline src/client/index.js:893]: just going to comment this once though it's literally the same change to different messages in this file. these rewards take a wallet address and a `denomToClaim` which is represented in the example above. usdxMinting claims and usdxMinting validator vesting claims do not follow this pattern, but hard, swap, and delegator related claims do

File: src/msg/kava/index.js [hunk 2]
```diff
@@ -191,70 +191,64 @@ function newMsgClaimUSDXMintingRewardVVesting(sender, receiver, multiplierName)
   }
 }
 
-function newMsgClaimHardReward(sender, multiplierName, denomsToClaim) {
+function newMsgClaimHardReward(sender, denomsToClaim) {
```
Comment (downing034) [inline src/msg/kava/index.js:194]: remove multiplierName from each msg builder function

File: src/msg/swap/index.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-function newMsgDeposit(despositor, tokenA, tokenB, slippage, deadline) {
+function newMsgDeposit(depositor, tokenA, tokenB, slippage, deadline) {
```
Comment (downing034) [inline src/msg/swap/index.js:1]: fix a typo

**Review by thomasmost — APPROVED** [review]
Looks good to me, though I have some questions to discuss live

File: src/msg/kava/index.js (hunk 2 above)
Comment (downing034) [reply src/msg/kava/index.js:194]: multiplier name is still used, it's just included in the denomsToClaim now.  Users are able to select a claim factor which are set in the incentive/parameters endpoint. The user then gets to choose between a large and small (currently) rewards factor and we multiply their wards value times this factor.

**Pull request merged**